pages/[channel]/[locale]/cart.tsx

```diff
@@ -8,6 +8,8 @@ import { messages } from "@/components/translations";
 import { usePaths } from "@/lib/paths";
 import { useCheckout } from "@/lib/providers/CheckoutProvider";
 
+const externalCheckoutUrl = process.env.NEXT_PUBLIC_CHECKOUT_URL;
```

Comment:
Please add `NEXT_PUBLIC_CHECKOUT_URL` to the `.env` file. I treat it as a single source of all env configurable options